ros2cli/ros2cli/command/daemon.py

```diff
@@ -22,11 +22,18 @@ class DaemonCommand(CommandExtension):
     """Various daemon related sub-commands."""
 
     def add_arguments(self, parser, cli_name):
+        self._subparser = parser
         # get verb extensions and let them add their arguments
         verb_extensions = get_verb_extensions('ros2cli.daemon.verb')
-        add_subparsers(parser, cli_name, '_verb', verb_extensions)
+        add_subparsers(
+            parser, cli_name, '_verb', verb_extensions, required=False)
 
     def main(self, *, parser, args):
+        if not hasattr(args, '_verb'):
+            # in case no verb was passed
+            self._subparser.print_help()
+            return 0
+
         # the attribute should always exist
         # otherwise argparse should have exited
```

Comment:
I don't think this comment is necessary anymore as the attribute existence is checked just above (same everywhere below)

Reply:
Removed the obsolete comments.